probe-rs/src/probe/blackmagic/mod.rs

```diff
@@ -1275,13 +1275,22 @@ impl RawProtocolIo for BlackMagicProbe {
 
     fn swj_pins(
         &mut self,
-        _pin_out: u32,
-        _pin_select: u32,
+        pin_out: u32,
```

Comment:
Ultimately we should abstract this interface to be usable in sequences. This was only introduced initially so we can port sequences from packs with ease.